Documentation/doc/scripts/process_doc.sh

```diff
@@ -52,13 +52,17 @@ cd $PATH_TO_SCRIPTS #scripts
 PATH_TO_MASTER="$PWD/doxygen_master/build/bin/doxygen"
 echo "done."
 
-echo "comparing versions 1.8.4 and 1.8.13"
-bash -$- test_doxygen_versions.sh $PATH_TO_1_8_4 $PATH_TO_1_8_13 $PWD/doc_1_8_4 $PWD/doc_1_8_13 $PUBLISH_DIR
+echo "comparing versions 1.8.18 and 1.8.13"
+bash -$- test_doxygen_versions.sh $PATH_TO_1_8_13 $PATH_TO_1_8_18 $PWD/doc_1_8_13 $PWD/doc_1_8_18  $PUBLISH_DIR
+if [ ! -d $PWD/doc_1_8_18/doc_log ]; then
```

Comment:
does it mean that `doc_1_8_13/doc_log` always if it exists for 1.8.18 ?

Reply:
yes